protosocket-connection/src/connection.rs

```diff
@@ -56,8 +58,26 @@ impl<Bindings: ConnectionBindings> Unpin for Connection<Bindings> {}
 impl<Bindings: ConnectionBindings> Future for Connection<Bindings> {
     type Output = ();
 
+    /**
+     * The poll method is the core of the Connection's asynchronous behavior.
+     * It's recommended to familiarize yourself with the ConnectionBindings trait
+     * in the types.rs file of this crate before proceeding.
```

Comment:
```suggestion
     * The poll method is the core of the Connection's asynchronous behavior.
     * It's recommended to familiarize yourself with the ConnectionBindings trait
     * in the types.rs file of this crate before proceeding.
```

I am not sure these are helpful lines.
* This is an implementation of std::future::Future: It is intrinsically the implementation of asynchronous behavior, because that is what std::future::Future is!
* Maybe you want to soften to "take a look at `ConnectionBindings` for the type definitions used by the Connection"